CHANGELOG.md

```diff
@@ -17,6 +17,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Datumaro no longer depends on scikit-image
   (<https://github.com/openvinotoolkit/datumaro/pull/379>)
 - `Dataset` remembers export options on saving / exporting for the first time (<https://github.com/openvinotoolkit/datumaro/pull/386>)
+- When using the OpenCV backend, Datumaro now saves images
```

Comment:
Probably, lines in changelog should be terser.

Reply:
Well... any suggestions for how to improve it? It's as terse as I could make it while also communicating all relevant information.

Reply:
It is a great explanation for the PR description field, but not a changelog entry.